04-cs-fundamentals/internship/exercises/binary-tree.md

```diff
@@ -0,0 +1,67 @@
+# Binary Tree Exercises
+
+## Instructions
+
+In pairs answer the following questions as if they are whiteboarding exercises.  Write down your answers for each step.
+
+For Each Question:
+
+- **Step 1:**  Write down 2 - 3 clarifying questions you would ask an interviewer.
```

Comment:
Should these line up with PSEs?

Reply:
Mostly they do, there are fewer clarifying questions because largely C11 is pretty good at that, and coming up with edge cases to test their code against, is something they struggle more with.